### PR TITLE
PRIVACY POLICY

### PR DESCRIPTION
**Pull request by marroe01284**
FEAT:
added h1 header
added section titles
added section content
added link to button -> ref new user
added link to mail adress(adress currently not working) STYLE:
added CSS styles to content following design
added media for tablet and desktop

### CODE REVIEW

**Comment by marroe01284** [comment]
FIX: 
when the footer is created for mobile and desktop it will be added to the page.

File: privacy.html [hunk 1]
```diff
@@ -14,7 +14,55 @@
...
+    <p class="section-content">We may update this privacy policy from time to time in order to reflect, for example,
+        changes to our practices or for other operational, legal or regulatory reasons.</p>
+    <h2 class="section-title">Contact Us</h2>
+    <p class="section-content">For more information about our privacy practices, if you have questions, or if you would
+        like to make a complaint, please contact us by email at <a href="#">leikr.contact@outlook.com</a>.</p>
+    <div class="btn-container">
+
+        <a href="newuser.html">
+            <button class="btn-primary">back</button>
+        </a>
```
Comment (KjetilHHauger) [inline privacy.html:64]: Wrong use of href for a button. make button with DIV

File: privacy.html [hunk 2]
```diff
@@ -14,7 +14,55 @@
 </head>
 
 <body>
+    <div>
+        <h1 class="section-header">Privacy Policy</h1>
+    </div>
+    <div class="container">
```
Comment (KjetilHHauger) [inline privacy.html:21]: Why spacing?

File: privacy.html [hunk 3]
```diff
@@ -14,7 +14,55 @@
...
 
+        <div class="header-line">
+
```
Comment (KjetilHHauger) [inline privacy.html:23]: Why spacing?

File: privacy.html [hunk 4]
```diff
@@ -14,7 +14,55 @@
...
+    <p class="section-content">When you place an order through Leikr, we will maintain your Order Information for our
+        records unless and until you ask us to delete this information.</p>
+    <h2 class="section-title">Changes</h2>
+    <p class="section-content">We may update this privacy policy from time to time in order to reflect, for example,
+        changes to our practices or for other operational, legal or regulatory reasons.</p>
+    <h2 class="section-title">Contact Us</h2>
+    <p class="section-content">For more information about our privacy practices, if you have questions, or if you would
+        like to make a complaint, please contact us by email at <a href="#">leikr.contact@outlook.com</a>.</p>
+    <div class="btn-container">
+
```
Comment (KjetilHHauger) [inline privacy.html:61]: Why spacing?

File: privacy.html [hunk 5]
```diff
@@ -14,7 +14,55 @@
...
+        <a href="newuser.html">
+            <button class="btn-primary">back</button>
+        </a>
+    </div>
 </body>
```
Comment (KjetilHHauger) [inline privacy.html:67]: Why spacing?

File: css/privacy.css [hunk 6]
```diff
@@ -3,4 +3,105 @@
...
+    color: var(--primary-color);
+}
+
+.btn-primary:hover {
+    background-color: var(--primary-color);
+    border: solid 1px var(--font-color);
+    color: var(--font-color);
+}
+
+
```
Comment (KjetilHHauger) [inline css/privacy.css:65]: Double spacing

**Review by TomChrister — APPROVED** [review]
Fantastisk

**Review by KjetilHHauger — APPROVED** [review]

**Pull request merged**